modcc/parser.cpp

```diff
@@ -782,6 +786,31 @@ std::pair<Token, Token> Parser::range_description() {
     return {lb, ub};
 }
 
+std::pair<Token, Token> Parser::from_to_description() {
+    Token lb, ub;
+
+    if(token_.type != tok::from) {
+        error(pprintf("range description must start with a FROM '%'", token_));
+        return {};
+    }
+
+    get_token();
+    lb = token_;
```

Comment:
Test that `token_` is a value?

Reply:
Good idea. We should add this to <lo, hi> ranges as well.